pkg/service/ec2/addresses.go

```diff
@@ -40,6 +40,8 @@ func (fn *DescribeAddresses) New(name string, config interface{}) ([]api.Request
 		if err != nil {
 			return err
 		}
+
+		// TODO: not noticing return value, but does it matter?
```

Comment:
@jta it returns right after this, but would it help to return an error instead? I'm having some trouble tracing how this actually gets called

Reply:
If you look at `api.SendRecords`, it returns `false` if `<-ctx.Done()`. So I think the correct behavior here would be to return `ctx.Err()` in that case.

Reply:
cool, that's what I was thinking—thanks for validating